ui/pages/Public/components/Faq.jsx

```diff
@@ -1,171 +1,503 @@
-    load data to seqr, you will need:
-
-    <List bulleted>
-      {WORKSPACE_REQUIREMENTS.map(item => <List.Item>{item}</List.Item>)}
-    </List>
-
-    If you do not have sufficient permissions on the workspace to request loading, you can contact the existing Owner of
-    the workspace to request for these permissions. Another option is to clone the existing workspace and then request
-    loading from the copy, as you will now be the Owner of the cloned workspace.
-
-    <Header content="Q. How much does it cost to use seqr?" size="medium" />
-    There are currently no costs associated with requests to load data from your AnVIL workspace to seqr or to use seqr
-    in AnVIL to analyze genomic data. <br /><br />
-
-    You will be responsible for the costs of storing your VCFs in an AnVIL workspace and will be responsible for any
-    compute operations you choose to run in that workspace, including the cost for generating any joint called VCFs.
-    You can find detailed information on AnVIL costs and billing &nbsp;
-    <a href="https://support.terra.bio/hc/en-us/articles/360048632271-Overview-Terra-costs-and-billing-GCP-" target="blank">here</a>.
-    Once it is confirmed that the data is accessible in seqr, the VCF can be removed from the AnVIL workspace.
-
-    <Header content="Q. How do I add data to an existing project in seqr?" size="medium" />
-    To add new data, create a new joint called VCF with all the samples you want in your seqr project, including those
-    you had previously loaded, and upload it using the Load Additional Data feature on the Project Page. All notes and
-    tags saved in previously analyzed cases will be kept.
-
-    <Header content="Q. How do I transfer data between workspaces in seqr?" size="medium" />
-    Each project in seqr is linked to a single workspace. We are unable to support loading a VCF from a new workspace
-    to an existing seqr project. <br /><br />
-
-    You can request to load a joint called VCF from a new workspace, which will create a new seqr project with the data.
-    None of your previous analysis such as tags and notes will be available in this new project.
-    Alternatively, if you want to add data to the existing project, you will need to move the new joint called VCF to
-    the original workspace and request loading additional data from that project as described above.
-
-    <Header content="Q. Have ideas for seqr?" size="medium" />
-    We are excited to see seqr&apos;s features grow to support your and others analysis needs, and welcome your
-    suggestions or code contributions to the open source project. Please open a new  &nbsp;
-    <a href="https://github.com/broadinstitute/seqr/discussions" target="blank">Github discussion</a> to discuss your
-    proposed ideas, or email us at <a href="mailto:seqr@broadinstitute.org">seqr@broadinstitute.org</a>.
+const FAQS = [
+  {
```

Comment:
Is it possible/reasonable for us to just change the `faq` route to redirect to `faq/en` or `faq/en-us` so that the object structure is consistent?

Reply:
its possible, but we do not do it anywhere else and its a bit of a pain with the combination of server side and client side routing we use in seqr. That said, changing the object structure makes sense to me and would be more readable, but we can acheive that without changing the routing and just setting a default for the language when its not in the route

Reply:
👍 let's do that then!